data-raw/example_downward_corrections_mat.R

```diff
@@ -25,60 +22,5 @@ example_downward_corr_mat <- matrix(
   byrow = TRUE
 )
 
-# Verify that this produces a PMF with negative entries
```

Comment:
Confirming that these can be removed because they are already tested in the examples (not in the tests from what I can see but thats fine)